ethpm_cli/commands/registry.py

```diff
@@ -107,6 +112,25 @@ def activate_registry(uri_or_alias: str, config: Config) -> None:
         write_store_data_to_disk(activated_store_data, store_path)
 
 
+def explore_registry(uri: str, config: Config) -> None:
+    if is_valid_registry_uri(URI(uri)):
+        (
+            address,
+            chain_id,
+            pkg_name,
+            pkg_version,
+            namespaced_asset,
+            ens,
+        ) = parse_registry_uri(uri)
```

Comment:
We shouldn't need any of these other than the `address` but was unable to find a suitable function, I'll use underscore to remove these unused variables